README.md

```diff
@@ -82,6 +82,13 @@ qemu-system-x86_64 -kernel path/to/kernel -initrd /tmp/initramfs.linux_amd64.cpi
 Note that you do not have to build a special kernel on your own, it is
 sufficient to use an existing one. Usually you can find one in `/boot`.
 
+>NOTE: you can compress the initramfs but for xz compression, the kernel has some restrictions
```

Comment:
can you take this part out and put it in a different commit with a message? It's best to limit commits to a single subject. Thanks.

Reply:
That was my intent. I am new to github PR process. So I naively thought creating a second pull request would be targeted at the last commit only... will do.